tools/device-farm-runner/run_perf_workflow.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+
+import copy
+import datetime
```

Comment:
Are these used anywhere?  If not, let's remove these and any other unused imports